tf-deployer/deploy/deploy.go

```diff
@@ -47,6 +48,8 @@ func NewDeployer(configPath, namedMachinesPath string) *deployer {
 		configPath = os.Getenv("KUBECONFIG")
 		if configPath == "" {
 			configPath = apiutil.GetDefaultKubeConfigPath()
+		} else {
+			configPath = strings.Split(configPath, ":")[0]
```

Comment:
Why this? (+ add a comment explaining the same)

Reply:
KUBECONIFG can be support more than one path.
https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#set-the-kubeconfig-environment-variable

Reply:
Then why choose the first one? (also add a comment for the same).